src/core/Text/Subtitle/styles.ts

```diff
@@ -0,0 +1,22 @@
+import { makeStyles, Theme } from '@material-ui/core';
+
+interface StyleProps {
+  color?: string;
+}
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    color: (props: StyleProps) =>
+      props ? props.color : theme.palette.text.primary,
+  },
+  primary: {
+    fontSize: '1rem',
+    fontWeight: 'normal',
+  },
+  small: {
+    fontSize: '0.75rem',
+    fontWeight: 'normal',
+  },
```

Comment:
This style hasn't been updated like the others yet. Could you please make this change!